src/server/command_registry.h

```diff
@@ -227,16 +217,10 @@ class CommandRegistry {
   std::pair<const CommandId*, facade::ArgSlice> FindExtended(std::string_view cmd,
                                                              facade::ArgSlice tail_args) const;
 
-  bool IsAlias(std::string_view cmd) const;
-
  private:
   absl::flat_hash_map<std::string, CommandId> cmd_map_;
   absl::flat_hash_map<std::string, std::string> cmd_rename_map_;
-  // Stores a mapping from alias to original command. During the find operation, the first lookup is
-  // done in the cmd_map_, then in the alias map. This results in two lookups but only for commands
-  // which are not in original map, ie either typos or aliases. While it would be faster, we cannot
-  // store iterators into cmd_map_ here as they may be invalidated on rehashing.
-  absl::flat_hash_map<std::string, std::string> cmd_aliases_;
+  absl::flat_hash_map<std::string, std::string> alias_map_;
```

Comment:
why do you need it as a class variable?

Reply:
I moved the map to a static variable and removed from class, I did have to add a helper to reset the map, as otherwise the tests fail because the map is initialized only once in other tests and by the time alias unit test runs the map is not updated.

Reply:
if we move the logic to Init function - it could be local variable. Let's see if we can solve the ACL stuff

Reply:
moved everything to init function and local variable derived from absl flag